collect_app/src/main/java/org/odk/collect/android/activities/FillBlankFormActivity.kt

```diff
@@ -218,11 +197,15 @@ class FillBlankFormActivity :
     private fun onMapButtonClick(id: Long) {
         permissionsProvider.requestLocationPermissions(
             this,
-            object : PermissionListener {
+            object :
+                PermissionListener {
```

Comment:
We don't have any check for this but to me it's weird to have that in two lines I would rather have: 
`object : PermissionListener {`